integration/matchers/matchers.go

```diff
@@ -73,28 +71,28 @@ func (m *existingCluster) Match(actual interface{}) (success bool, err error) {
 		return false, errors.New("input is nil")
 	}
 
-	if reflect.TypeOf(actual).String() != "*session.Session" {
+	if reflect.TypeOf(actual).String() != "*aws.Config" {
 		return false, errors.New("not a AWS session")
 	}
 
-	eks := awseks.New(actual.(*session.Session))
+	client := eks.NewFromConfig(actual.(aws.Config))
 
-	input := &awseks.DescribeClusterInput{
+	input := &eks.DescribeClusterInput{
 		Name: aws.String(m.expectedName),
 	}
-	output, err := eks.DescribeCluster(input)
+	output, err := client.DescribeCluster(context.Background(), input)
 
 	if err != nil {
-		// Check if its a not found error: ResourceNotFoundException
-		if !strings.Contains(err.Error(), awseks.ErrCodeResourceNotFoundException) {
+		// Check if it's a not found error: ResourceNotFoundException
+		if !strings.Contains(err.Error(), "ResourceNotFoundException") {
```

Comment:
A more idiomatic approach (requires importing `aws-sdk-go-v2/service/eks/types`):

```suggestion
		var notFoundErr *ekstypes.ResourceNotFoundException
		if !errors.As(err, &notFoundErr) {
```

Reply:
Yeah I was lazy hahahahhaa, thanks. Also I need to run integration tests on this branch. :D